src/main/helpers/controller/ContactTribunalSelectedHelper.ts

```diff
@@ -0,0 +1,35 @@
+import { CaseWithId } from '../../definitions/case';
+import { PageUrls, ValidationErrors } from '../../definitions/constants';
+import { Application, isTypeAOrB } from '../../definitions/contact-tribunal-applications';
+import { FormError } from '../../definitions/form';
+import { isContentCharsOrLess, isFieldFilledIn } from '../../validators/validator';
+
+export const getNextPage = (app: Application, userCase: CaseWithId): string => {
+  return isTypeAOrB(app) ? getNextCopyPage(userCase) : PageUrls.CONTACT_TRIBUNAL_CYA;
+};
+
+const getNextCopyPage = (userCase: CaseWithId): string => {
+  return isClaimantSystemUser(userCase) ? PageUrls.COPY_TO_OTHER_PARTY : PageUrls.COPY_TO_OTHER_PARTY_OFFLINE;
```

Comment:
@cindychmcts we'd also need to check if the claim was submitted through MyHMCTS as those would also be counted as online claims

Reply:
can add as a todo but thought I'd mention